src/libs/histogram.c

```diff
@@ -155,17 +166,18 @@ const char **views(dt_lib_module_t *self)
 
 uint32_t container(dt_lib_module_t *self)
 {
-  return DT_UI_CONTAINER_PANEL_RIGHT_TOP;
+  const dt_lib_histogram_position_t position = dt_conf_get_int("plugins/darkroom/histogram/position");
+  return position;
```

Comment:
`return position == 0 ?DT_UI_CONTAINER_PANEL_LEFT_TOP :  DT_UI_CONTAINER_PANEL_RIGHT_TOP;`

Because DT_UI_CONTAINER_PANEL_RIGHT_TOP is 3.